### ISSUE TITLE
Neovide does not start

### ISSUE THREAD

**Issue by cassis163** [neovide/neovide#1663]
**Describe the bug**
Neovide is installed, but does not start if I run `neovide` in the terminal.

**To Reproduce**
1. Install neovide via cargo
2. Add `~/.cargo/bin/neovide` to PATH via `.zshrc`
3. Run neovide

**Expected behavior**
I would expect neovide to start.

**Desktop (please complete the following information):**
 - OS: Fedora 37 (Gnome + Wayland)
 - Neovide Version 0.10.3
 - Neovim Version 0.8.1

**Please run `neovide --log` and paste the contents of the `.log` file created in the current directory here:**
Neovide crashes after:
```
TRACE [neovide::redraw_scheduler] Next frame queued
TRACE [neovide::settings] Initial value load failed for cursor_distance_length_adjust: Error processing request: 1 - 'Key not found: neovide_cursor_distance_length_adjust')
TRACE [neovide::settings] Initial value load failed for cursor_animation_length: Error processing request: 1 - 'Key not found: neovide_cursor_animation_length')
TRACE [neovide::settings] Initial value load failed for remember_window_size: Error processing request: 1 - 'Key not found: neovide_remember_window_size')
TRACE [neovide::settings] Initial value load failed for padding_right: Error processing request: 1 - 'Key not found: neovide_padding_right')
TRACE [neovide::settings] Initial value load failed for input_use_logo: Error processing request: 1 - 'Key not found: neovide_input_use_logo')
TRACE [neovide::settings] Initial value load failed for padding_bottom: Error processing request: 1 - 'Key not found: neovide_padding_bottom')
TRACE [neovide::settings] Initial value load failed for scale_factor: Error processing request: 1 - 'Key not found: neovide_scale_factor')
TRACE [neovide::settings] Initial value load failed for position_animation_length: Error processing request: 1 - 'Key not found: neovide_position_animation_length')
TRACE [neovide::settings] Initial value load failed for cursor_vfx_particle_lifetime: Error processing request: 1 - 'Key not found: neovide_cursor_vfx_particle_lifetime')
TRACE [neovide::settings] Initial value load failed for cursor_trail_size: Error processing request: 1 - 'Key not found: neovide_cursor_trail_size')
TRACE [neovide::settings] Initial value load failed for background_color: Error processing request: 1 - 'Key not found: neovide_background_color')
TRACE [neovide::settings] Initial value load failed for cursor_vfx_mode: Error processing request: 1 - 'Key not found: neovide_cursor_vfx_mode')
TRACE [neovide::settings] Initial value load failed for cursor_vfx_particle_density: Error processing request: 1 - 'Key not found: neovide_cursor_vfx_particle_density')
TRACE [neovide::settings] Initial value load failed for refresh_rate_idle: Error processing request: 1 - 'Key not found: neovide_refresh_rate_idle')
TRACE [neovide::settings] Initial value load failed for fullscreen: Error processing request: 1 - 'Key not found: neovide_fullscreen')
TRACE [neovide::settings] Initial value load failed for cursor_antialiasing: Error processing request: 1 - 'Key not found: neovide_cursor_antialiasing')
TRACE [neovide::settings] Initial value load failed for cursor_vfx_particle_speed: Error processing request: 1 - 'Key not found: neovide_cursor_vfx_particle_speed')
TRACE [neovide::settings] Initial value load failed for input_macos_alt_is_meta: Error processing request: 1 - 'Key not found: neovide_input_macos_alt_is_meta')
TRACE [neovide::settings] Initial value load failed for refresh_rate: Error processing request: 1 - 'Key not found: neovide_refresh_rate')
TRACE [neovide::settings] Initial value load failed for hide_mouse_when_typing: Error processing request: 1 - 'Key not found: neovide_hide_mouse_when_typing')
TRACE [neovide::settings] Initial value load failed for padding_top: Error processing request: 1 - 'Key not found: neovide_padding_top')
TRACE [neovide::settings] Initial value load failed for cursor_unfocused_outline_width: Error processing request: 1 - 'Key not found: neovide_cursor_unfocused_outline_width')
TRACE [neovide::settings] Initial value load failed for confirm_quit: Error processing request: 1 - 'Key not found: neovide_confirm_quit')
TRACE [neovide::settings] Initial value load failed for transparency: Error processing request: 1 - 'Key not found: neovide_transparency')
TRACE [neovide::settings] Initial value load failed for debug_renderer: Error processing request: 1 - 'Key not found: neovide_debug_renderer')
TRACE [neovide::settings] Initial value load failed for underline_automatic_scaling: Error processing request: 1 - 'Key not found: neovide_underline_automatic_scaling')
TRACE [neovide::settings] Initial value load failed for no_idle: Error processing request: 1 - 'Key not found: neovide_no_idle')
TRACE [neovide::settings] Initial value load failed for touch_drag_timeout: Error processing request: 1 - 'Key not found: neovide_touch_drag_timeout')
TRACE [neovide::settings] Initial value load failed for floating_blur_amount_y: Error processing request: 1 - 'Key not found: neovide_floating_blur_amount_y')
TRACE [neovide::settings] Initial value load failed for profiler: Error processing request: 1 - 'Key not found: neovide_profiler')
TRACE [neovide::settings] Initial value load failed for cursor_animate_command_line: Error processing request: 1 - 'Key not found: neovide_cursor_animate_command_line')
TRACE [neovide::settings] Initial value load failed for cursor_vfx_particle_phase: Error processing request: 1 - 'Key not found: neovide_cursor_vfx_particle_phase')
TRACE [neovide::settings] Initial value load failed for cursor_vfx_particle_curl: Error processing request: 1 - 'Key not found: neovide_cursor_vfx_particle_curl')
TRACE [neovide::settings] Initial value load failed for iso_layout: Error processing request: 1 - 'Key not found: neovide_iso_layout')
TRACE [neovide::settings] Initial value load failed for touch_deadzone: Error processing request: 1 - 'Key not found: neovide_touch_deadzone')
TRACE [neovide::settings] Initial value load failed for remember_window_position: Error processing request: 1 - 'Key not found: neovide_remember_window_position')
TRACE [neovide::settings] Initial value load failed for scroll_animation_length: Error processing request: 1 - 'Key not found: neovide_scroll_animation_length')
TRACE [neovide::settings] Initial value load failed for floating_opacity: Error processing request: 1 - 'Key not found: neovide_floating_opacity')
TRACE [neovide::settings] Initial value load failed for floating_blur_amount_x: Error processing request: 1 - 'Key not found: neovide_floating_blur_amount_x')
TRACE [neovide::settings] Initial value load failed for floating_blur: Error processing request: 1 - 'Key not found: neovide_floating_blur')
TRACE [neovide::settings] Initial value load failed for cursor_vfx_opacity: Error processing request: 1 - 'Key not found: neovide_cursor_vfx_opacity')
TRACE [neovide::settings] Initial value load failed for padding_left: Error processing request: 1 - 'Key not found: neovide_padding_left')
TRACE [neovide::settings] Initial value load failed for cursor_animate_in_insert_mode: Error processing request: 1 - 'Key not found: neovide_cursor_animate_in_insert_mode')
```

**Comment by Ouilly** [comment]
I am also unable to start Neovide from the applications menu of Manjaro, would be curious to see if the issues could have a similar cause!

**Comment by alexs-sh** [comment]
@Ouilly Hi, are you using Gnome + Wayland too?

**Comment by alexs-sh** [comment]
I tried to reproduce the issue. For now, everything is fine. At least NVIM v0.8.1 + neovide 0.10.3 works fine with Xorg + awesomewm / xfce. But I'm going to use Gnome + Wayland to repeat the environment

**Comment by Ouilly** [comment]
@alexs-sh Hi, yes I am!

**Comment by alexs-sh** [comment]
@Ouilly @cassis163 I tried neovide at Fedora 37 with Gnome and Wayland. It still works.
![image](https://user-images.githubusercontent.com/50661022/205654789-77abc505-e64a-41e9-b927-ca098af669d8.png)

Does running without a custom config changes something?

```
neovide -- -u none
```

**Comment by kdarkhan** [comment]
Probably a duplicate of https://github.com/neovide/neovide/issues/1356

**Comment by MultisampledNight** [comment]
Hi! For those with the original bug report, could you test from main if this is still an issue? The logs suggest otherwise, but I'm unsure.

**Comment by fredizzimo** [comment]
I'm pretty sure this is fixed in Neovide 0.11.0
